CONTRIBUTING.md

```diff
@@ -8,11 +8,25 @@ While we encourage all contributions, remember that Adamant is used everyday to
 
 ## Creating a Submission
 
-Adamant follows the [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow) development process. You should begin by [forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo) this repository. When you have finished making changes on your fork, create a pull request back to this repository.
+Adamant follows the [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow) development process. You should begin by [forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo) this repository. When you have finished making changes on your fork, create a [pull request](https://github.com/lasp/adamant/pulls) back to this repository.
 
-Please try to keep submissions small and focused on single fixes, enhancements, or additions. This makes changes easier to review and faster to merge.
+### Pull Request
 
-Once a pull request has been submitted the following will occur:
+Please adhere to the following when submitting a pull request:
+
+- Pull requests should be made against `main`.
```

Comment:
Is there a process that works for setting up multiple contributors on a single PR and is this worth mentioning here?

Reply:
Similar to my other comment. Since that is the exception, not the rule, it is hard to address with a single process IMO. But you are right, rebasing is problematic when more than one person is committing to a PR.